website/docs/en/config/server/https.mdx

```diff
@@ -23,17 +23,18 @@ HTTPS:
 
 You can manually pass in the certificate and the private key required in the `server.https` option. This parameter will be directly passed to the `createServer` method of the https module in Node.js.
 
+For basic configuration needs, you can add @rsbuild/plugin-basic-ssl to your project plugins, which will automatically create and cache a self-signed certificate. However, we recommend creating and using your own certificate.
+
 For details, please refer to [https.createServer](https://nodejs.org/api/https.html#https_https_createserver_options_requestlistener).
 
 ```ts
 import fs from 'node:fs';
+import { pluginBasicSsl } from '@rsbuild/plugin-basic-ssl';
 
 export default {
+  plugins: [pluginBasicSsl()],
   server: {
-    https: {
-      key: fs.readFileSync('certificates/private.pem'),
-      cert: fs.readFileSync('certificates/public.pem'),
-    },
+    https: {},
```

Comment:
Maybe we do not need users to set `server.https`.

Just register the plugin and HTTPS will work.

Reply:
Nice idea.